src/noit_filters.c

```diff
@@ -435,27 +443,7 @@ noit_apply_filterrule_metric(filterrule_t *r,
   }
   return mtev_true;
 }
-static int
-noit_filter_update_conf_rule(const char *fname, int idx, const char *rname, const char *value) {
-  char xpath[1024];
-  mtev_conf_section_t rulenode;
-  xmlNodePtr child;
 
-  snprintf(xpath, sizeof(xpath), "//filtersets//filterset[@name=\"%s\"]/rule[%d]", fname, idx);
-  rulenode = mtev_conf_get_section(MTEV_CONF_ROOT, xpath);
-  if(mtev_conf_section_is_empty(rulenode)) {
-    mtev_conf_release_section(rulenode);
-    return -1;
-  }
-  child = xmlNewNode(NULL, (xmlChar *)rname);
-  xmlNodeAddContent(child, (xmlChar *)value);
-  xmlAddChild(mtev_conf_section_to_xmlnodeptr(rulenode), child);
-  CONF_DIRTY(rulenode);
-  mtev_conf_mark_changed();
-  mtev_conf_request_write();
-  mtev_conf_release_section(rulenode);
-  return 0;
-}
```

Comment:
This was just saving the names of metrics that were matched?  i.e. not part of how the filterset definition gets saved?

Reply:
It was saving, but the saving had a bug. I fixed the bug and now saving is to slow to be useful. It also turns out that saving isn't really what we want. We want show churn.